src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -4,322 +4,150 @@ tags:
   - Synthetics
   - Synthetic monitoring
   - Getting started
+  - Synthetic monitoring tutorial
+  - Synthetics tutorial
 translate:
   - jp
-metaDescription: "New Relic's synthetic monitoring capabilities give you automated, scriptable tools to monitor your websites, critical business transactions, and API endpoints."
+metaDescription: "Get started with New Relic by creating your first simple browser or scripted browser monitor."
 redirects:
   - /docs/new-relic-only/new-relic-synthetics/getting-started/new-relic-synthetics
   - /docs/synthetics/new-relic-synthetics/getting-started/new-relic-synthetics
   - /docs/synthetics/new-relic-synthetics/getting-started/introduction-new-relic-synthetics
   - /docs/synthetics/synthetic-monitoring/getting-started/get-started-new-relic-synthetic-monitoring
   - /docs/synthetics
   - /docs/synthetics/synthetic-monitoring/getting-started
+  - /docs/synthetics/synthetic-monitoring/scripting-monitors/scripted-browser-examples
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
 ---
 
-You can think of our synthetic monitors as crash test dummies for your websites, applications, and API endpoints. When you deploy a synthetic monitor, an API check or virtual browser instance runs through your most important workflows, testing your critical endpoints and reporting the results to New Relic. If there's an error, failure, or anomaly affecting your customer-facing apps, then your synthetic monitor will catch it and alert you, all before anyone has any time to notice. 
+import syntheticResultsPageforSimpleBrowserMonitor from 'images/synthetic_screenshot-full_Results-page-for-simple-browser-monitor.png'
 
-Our synthetic monitoring capabilities work on your internal apps and behind firewalls, too. Set up a private location, then send containerized applications to manage the jobs (that is, to deploy the kind of synthetic monitor you'd like to deploy) in a secured context. 
+Have an application that needs to stay live? Synthetic monitoring can help you proactively catch and resolve issues with your apps before they affect your customers. Don't have an account, but want to create a synthetic monitor? Sign up for your free [New Relic account](https://newrelic.com/signup).
```

Comment:
I think you can shorten the last part to just "Sign up for free". It feels like we're using account twice in too short a span. Just a thought!

Reply:
TBH AGREE!